ee/codegen/src/generators/graph-attribute.ts

```diff
@@ -681,6 +660,42 @@ export class GraphAttribute extends AstNode {
     return mutableAst;
   }
 
+  /**
+   * Pops the source node from the AST, returning a new AST without the source node.
+   *
+   * Example:
+   *
+   * ```
+   * A >> B >> C
+   * ```
+   *
+   * Becomes:
+   *
+   * ```
+   * B >> C
+   * ```
+   */
+  private popSources = (mutableAst: GraphMutableAst): GraphMutableAst => {
+    if (mutableAst.type === "set") {
+      return this.flattenSet({
```

Comment:
This `this.flattenSet()` is the only actual change to the method to get the new test to pass